app/assets/javascripts/listeners.js

```diff
@@ -33,7 +33,7 @@ $(document).on('turbolinks:load', function() {
 });
 
 $(document).ready(function(){
-
+  // wpvi
```

Comment:
@cdoyle-temple I'm not familiar with wpvi.  What does this mean?

Reply:
This section is for the WPVI videos that we host in the site.

Reply:
I believe the alerts you look for in the JSON are Alert.where(published: true).where(for_header: false)

Nothing about that has changed.